recipes-bsp/imx-lib/imx-lib_git.bb

```diff
@@ -46,4 +46,4 @@ do_install () {
     find ${D}/ -name *.go -exec rm {} \;
 }
 
-COMPATIBLE_MACHINE = "(mx6-nxp-bsp|mx7-nxp-bsp|mx8ulp-nxp-bsp)"
+COMPATIBLE_MACHINE = "(mx6-nxp-bsp|mx7-nxp-bsp|mx8ulp-nxp-bsp|mx9-nxp-bsp)"
```

Comment:
Mind adding `PLATFORM:mx9-nxp-bsp = "IMX8ULP"` as we added mx9 to COMPATIBLE machines?